model/error/event_test.go

```diff
@@ -84,6 +84,9 @@ func TestErrorEventDecode(t *testing.T) {
 	name, userId, email, userIp := "jane", "abc123", "j@d.com", "127.0.0.1"
 	context := map[string]interface{}{"a": "b", "user": map[string]interface{}{
 		"username": name, "email": email, "ip": userIp, "id": userId}}
+	origContext := map[string]interface{}{"a": "b", "user": map[string]interface{}{
```

Comment:
because I remove the `tags` from context. If I simply check against the same object, that would match, no matter if the tags were actually removed or not.